.github/workflows/release.yml

```diff
@@ -0,0 +1,109 @@
+name: Release automation
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit_id:
+        description: 'Commit ID to tag and push to remote'
+        required: true
```

Comment:
can't we take the HEAD commit from release-candidate?

Reply:
It can be but no harm in explicitly providing the commit ID in-case a release-candidate approach is not used for releasing in the future.